lib/browsers.js

```diff
@@ -66,7 +66,12 @@ var browserDefinitions = {
     },
     opera: {
         regex: /Opera (\S+)/,
-        profile: true
+        profile: true,
+        variants: {
+            'opera': ['opera', 'opera-mini'],
```

Comment:
Does Opera Mini exist on desktop anywhere? I thought it was mobile only, and this library doesn't support mobile at all right now.

Great browser though, I used to use it _years_ ago (on Symbian, I think?) I haven't heard anybody mention it in ages! :smile:

Reply:
Yes, you are right, opera-mini is still only available for mobile (I thought they would have added desktop support by now). Removing it now, because it might be confusing to keep it here when it is actually not supported.